### PR TITLE
Angular signal props and BaseWidgetDirective upgrade

### PR DESCRIPTION
**Pull request by quentinderoubaix**

### CODE REVIEW

File: angular/bootstrap/src/components/progressbar/progressbar.component.ts [hunk 1]
```diff
@@ -159,19 +154,22 @@ export class ProgressbarComponent extends BaseWidgetDirective<ProgressbarWidget>
...
+
+	constructor() {
+		super(
+			callWidgetFactory({
+				factory: createProgressbar,
+				widgetName: 'progressbar',
+				defaultConfig,
+				afterInit: (widget) => useDirectiveForHost(widget.directives.ariaDirective),
+				slotTemplates: () => ({
+					children: undefined,
```
Comment (andreascorti) [inline angular/bootstrap/src/components/progressbar/progressbar.component.ts:168]: as discussed, we should either filter out children, or (better) remove the possibility to have both a slot and a ng-content

**Review by andreascorti — COMMENTED** [review]
Thanks @quentinderoubaix ! some comments already discussed with you

File: angular/bootstrap/src/components/progressbar/progressbar.component.ts (hunk 1 above)
Comment (quentinderoubaix) [reply angular/bootstrap/src/components/progressbar/progressbar.component.ts]: as discussed, it's better to provide a directive for type checking ng-templates

File: angular/demo/bootstrap/src/app/samples/rating/config.route.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 import {Component, signal} from '@angular/core';
 
 @Component({
 	standalone: true,
-	imports: [RatingComponent],
+	imports: [RatingComponent, RatingStarDirective],
 	providers: [provideWidgetsConfig()],
 	template: `
-		<ng-template #custom let-fill="fill">
+		<ng-template #custom auRatingStar let-fill="fill">
```
Comment (andreascorti) [inline angular/demo/bootstrap/src/app/samples/rating/config.route.ts:10]: Question: is it mentioned somewhere in the documentation?

**Review by andreascorti — APPROVED** [review]
All comments and discussion have been addressed 
LGTM
Thanks @quentinderoubaix !

**Pull request merged**